docs/simreal/timesteplogic.rst

```diff
@@ -1,6 +1,6 @@
-***************************
-Timestep Logic
-***************************
+********************************
+"Real" and "Non-real" TimeSteps
```

Comment:
```suggestion
"Real" and "Non-real" Time Steps
```

Reply:
This title is not very clear to me.  What would be a non-real timestep?  Do you mean non-real-time?

Reply:
I think it sounds good... makes it clear the notion of time is different in simulation and reality, and then this section goes on to explain that